db/migrate/20230221162117_create_guest_rooms.rb

```diff
@@ -0,0 +1,10 @@
+class CreateGuestRooms < ActiveRecord::Migration[5.2]
+  def change
+    create_table :guest_rooms do |t|
+      t.references :guest, foreign_key: true
+      t.references :room, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Looking at this migration, I can almost say for certain that you are using the `rails g model` command. Kudos for picking that up! 🙌